Error compiling sample project called Example.NetFramework

It generates a lot of errors:
```
error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\Customers\customer.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\order.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\sample.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\security.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\sql\sample.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\Customers\customer.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\order.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\sample.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\security.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\sql\sample.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\bin\Debug\yesql\ThirdParties\thirdParty.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\Customers\customer.sql" because it was not found.
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(5198,5): error MSB3030: Could not copy the file "C:\Users\mrdav\OneDrive\Documentos\Visual Studio 2022\Projects\yesql.net\samples\Example.NetFramework\bin\Debug\yesql\bin\Debug\yesql\order.sql" because it was not found.
```
This is a very strange error, since msbuild should not look for any sql files from the bin folder, because they are excluded with the property called [DefaultItemExcludes](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#defaultitemexcludes). However, I have verified that this property is not set (returns an empty string) when the project points to .NET Framework instead of dotnetcore.

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -3,7 +3,7 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
   </PropertyGroup>
   <ItemGroup>
-    <PackageVersion Include="CopySqlFilesToOutputDirectory" Version="0.1.0-alpha" />
+    <PackageVersion Include="CopySqlFilesToOutputDirectory" Version="0.2.0-alpha" />
     <PackageVersion Include="CopySqlFilesToPublishDirectory" Version="0.1.0-alpha" />
     <PackageVersion Include="Microsoft.AspNetCore.OpenApi" Version="7.0.10" />
     <PackageVersion Include="Swashbuckle.AspNetCore" Version="6.5.0" />

--- a/props/YeSql.Net.nuspec
+++ b/props/YeSql.Net.nuspec
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
         <id>CopySqlFilesToOutputDirectory</id>
-        <version>0.1.0-alpha</version>
+        <version>0.2.0-alpha</version>
         <authors>OSE.NET</authors>
         <owners>OSE.NET</owners>
         <projectUrl>https://github.com/ose-net/yesql.net</projectUrl>

--- a/props/build/CopySqlFilesToOutputDirectory.props
+++ b/props/build/CopySqlFilesToOutputDirectory.props
@@ -6,7 +6,7 @@
       Other directories such as ./bin and ./obj are also excluded.
     -->
     <DefaultItemExcludes>
-      $(DefaultItemExcludes);out/**;publish/**
+      $(DefaultItemExcludes);out/**;publish/**;bin/**;obj/**
     </DefaultItemExcludes>
   </PropertyGroup>
 

--- a/samples/YeSql.Net.Examples.sln
+++ b/samples/YeSql.Net.Examples.sln
@@ -21,6 +21,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Example.NetFramework", "Exa
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{91D295C8-520D-46AF-95BC-D1E6C1E60534}"
 	ProjectSection(SolutionItems) = preProject
+		..\props\build\CopySqlFilesToOutputDirectory.props = ..\props\build\CopySqlFilesToOutputDirectory.props
+		..\targets\build\CopySqlFilesToPublishDirectory.targets = ..\targets\build\CopySqlFilesToPublishDirectory.targets
 		..\Directory.Build.props = ..\Directory.Build.props
 		..\Directory.Packages.props = ..\Directory.Packages.props
 	EndProjectSection

